### PR TITLE
`get_observatory` shouldn't overwrite `include_bipm` and `include_gps` unless given explicitly 

### PR DESCRIPTION
**Pull request by abhisrkckl**
#1691 is a symptom of this, although the root cause is elsewhere.

#1707 was an attempt at fixing this.

Related issue - #1708

### CODE REVIEW

File: src/pint/observatory/__init__.py [hunk 1]
```diff
@@ -143,24 +144,6 @@ class Observatory:
...
-                raise ValueError(
-                    f"Observatory {name.lower()} already present and overwrite=False"
-                )
-            log.warning(f"Observatory '{name.lower()}' already present; overwriting...")
-
-            cls._register(obs, name)
-            return obs
-        cls._register(obs, name)
-        return obs
-
```
Comment (abhisrkckl) [inline src/pint/observatory/__init__.py:163]: Everything here can be done in `__init__()`. Having a non-trivial `__new__()` makes the objects non-copyable. That's why I am moving this code to `__init__()`.

File: CHANGELOG-unreleased.md [hunk 2]
```diff
@@ -40,4 +40,5 @@ the released changes.
 - Consistent naming in `TimingModel.get_params_mapping()`
 - Better exceptions for unsupported/unimplemented binary models (BTX, MSS, etc.)
 - Emit warnings when `WaveX`/`DMWaveX` is used together with other representations of red/DM noise
+- `get_observatory()` no longer overwrites `include_gps` and `include_bipm` of `Observatory` objects unless explicitly stated.
```
Comment (paulray) [inline CHANGELOG-unreleased.md:43]: I think we should add "Fixes issue #1691 (BIPM+GPS clock corrections incorrectly applied to BAT TOAs)"

**Review by paulray — APPROVED** [review]
Looks good to me. BAT times should not get BIPM or GPS corrections applied, regardless of whether they are TZRMJD times or TOA times. BAT times are already fully corrected to TDB.

File: CHANGELOG-unreleased.md (hunk 2 above)
Comment (abhisrkckl) [reply CHANGELOG-unreleased.md]: Added.

**Comment by abhisrkckl** [comment]
@paulray Can you please merge this?

**Pull request merged**